### PR TITLE
Change standalone agency doc

### PR DESCRIPTION
**Pull request by lservini**
Changed documentation for standalone agency, set port of first agent to 8531, offset +10 for other agents for ArangoDB standard consistency

### CODE REVIEW

File: Documentation/Books/Manual/Deployment/StandaloneAgency/README.md [hunk 1]
```diff
@@ -22,12 +22,12 @@ Clearly all cases, which would form disjunct subsets of agents would break or in
 
 Typically, one achieves fairly high fault-tolerance with low, odd number of agents while keeping the necessary network traffic at a minimum. It seems that the typical agency size will be in range of 3 to 7 agents.
```
Comment (Simran-B) [inline Documentation/Books/Manual/Deployment/StandaloneAgency/README.md]: Maybe easier to list all three options, given that it should be an odd number. Is an even number even possible? The text doesn't clearly state that it *must* be an odd number...

```suggestion
Typically, one achieves fairly high fault-tolerance with low, odd number of agents while keeping the necessary network traffic at a minimum. It seems that the typical agency size will be 3, 5 or 7 agents.
```
Also, "it seems" sounds rather vague. How about saying "It should typically be ..."?

File: Documentation/Books/Manual/Deployment/StandaloneAgency/README.md [hunk 2]
```diff
@@ -22,12 +22,12 @@ Clearly all cases, which would form disjunct subsets of agents would break or in
...
 
-The below commands start up a 3-host agency on one physical/logical box with ports 8529, 8530 and 8531 for demonstration purposes. The address of the first instance, port 8529, is known to the other two. After at most 2 rounds of gossipping, the last 2 agents will have a complete picture of their surrounding and persist it for the next restart.
+The below commands start up a 3-host agency on one physical/logical box with ports 8531, 8541 and 8551 for demonstration purposes. The address of the first instance, port 8531, is known to the other two. After at most 2 rounds of gossipping, the last 2 agents will have a complete picture of their surrounding and persist it for the next restart.
```
Comment (Simran-B) [inline Documentation/Books/Manual/Deployment/StandaloneAgency/README.md]:
```suggestion
The below commands start up a 3-host agency on one physical/logical box with ports 8531, 8541 and 8551 for demonstration purposes. The address of the first instance, port 8531, is known to the other two. After at most 2 rounds of gossipping, the last 2 agents will have a complete picture of their surroundings and persist it for the next restart.
```

File: Documentation/Books/Manual/Deployment/StandaloneAgency/README.md [hunk 3]
```diff
@@ -22,12 +22,12 @@ Clearly all cases, which would form disjunct subsets of agents would break or in
...
-./arangod --agency.activate true --agency.size 3 --agency.endpoint tcp://localhost:8529 --agency.my-address tcp://localhost:8531 --server.authentication false --server.endpoint tcp://0.0.0.0:8531 agency-8531 
+./arangod --agency.activate true --agency.size 3 --agency.my-address tcp://localhost:8531 --server.authentication false --server.endpoint tcp://0.0.0.0:8531 agency-8531
+./arangod --agency.activate true --agency.size 3 --agency.my-address tcp://localhost:8541 --server.authentication false --server.endpoint tcp://0.0.0.0:8541 --agency.endpoint tcp://localhost:8531 agency-8541
+./arangod --agency.activate true --agency.size 3 --agency.my-address tcp://localhost:8551 --server.authentication false --server.endpoint tcp://0.0.0.0:8551 --agency.endpoint tcp://localhost:8531 agency-8551 
 ```
 
 The parameter `agency.endpoint` is the key ingredient for the second and third instances to find the first instance and thus form a complete agency. Please refer to the the shell-script `scripts/startStandaloneAgency.sh` on github or in the source directory.
```
Comment (Simran-B) [inline Documentation/Books/Manual/Deployment/StandaloneAgency/README.md]:
```suggestion
The parameter `agency.endpoint` is the key ingredient for the second and third instances to find the first instance and thus form a complete agency. Please refer to the the shell-script `scripts/startStandaloneAgency.sh` on GitHub or in the source directory.
```

**Comment by lservini** [comment]
@Simran-B 
-L does not redirect to Leader in my test. Just get the config from followers

**Comment by Simran-B** [comment]
I opened an issue in the new docs repo regarding the redirect question: https://github.com/arangodb/docs/issues/14

**Pull request merged**